app/controllers/posts_controller.rb

```diff
@@ -30,4 +30,13 @@ def create
       end
     end
   end
+
+  def destroy
+    @post = Post.find(params[:id])
```

Comment:
- [ ] Great job adding authentication from the view 👏🏻 But you need to add this `load_and_authorize_resource` as weel to the top of your Post Controller and the  Comment controller to make sure your authorization rules are enforceable and use **CanCanCan** for post and comment authorization 😉 👍🏻 

- Note: you could also add `authorize_resource` if you had this line eg;
`Before_action:set_name, only: %[show update destroy] ` at the top of your controller